hmt_escrow/job.py

```diff
@@ -442,6 +450,11 @@ def setup(self, gas: int = GAS_LIMIT) -> bool:
             self.manifest_hash,
         ]
 
+        if not hmt_transferred and blocking:
```

Comment:
this `if` condition is not going to be executed, because in [this line](https://github.com/humanprotocol/hmt-escrow/pull/275/files#diff-41bbe49a252d9374baea7976d28fdf0cbdf6037ec58e38dafe71e78036c8ee74R436) it is already checked(`if not hmt_transferred`)

Reply:
I think, here you meant `if hmt_transferred and blocking`

Reply:
if hmt_transfered, then I don't want to execute the code. Yeah, I agree, negations are always troublesome. But I don't want to introduce too many changes.

Reply:
here I see the next code:

```
# 1
if not hmt_transferred:
      return False
...
# 2 
if not hmt_transferred and blocking:
     # .. your code
```

and the `# 2` will not be executed because of the return statement in `# 1`

maybe I don't see something in here:)

Reply:
yes, that means that the contract was never created, so it is safe to skip the setup

Reply:
bah, you are right. the flag is contract_is_setup not hmt_transferred.

Reply:
Updating...